plugins-ui/src/components/PluginPolicyModal.tsx

```diff
@@ -325,7 +342,12 @@ export const PluginPolicyModal: FC<PluginPolicyModalProps> = ({
                             name={parameterName}
                             rules={[{ required }]}
                           >
-                            <Stack as={Input} $style={{ height: "44px" }} />
```

Comment:
We can simplified  by just

`<Stack as={Input} $style={{ height: "44px" }} disabled={isFeesPlugin} />`